Assets/_Functionalities/Wms/Scripts/ImageProjectionLayer.cs

```diff
@@ -155,6 +162,80 @@ protected virtual Tile CreateNewTile(Vector2Int tileKey)
             return tile;
         }
 
+        public void RefreshTiles()
+        {
+            //is the update already running cancel it
+            if (visibleTilesDirty && updateTilesRoutine != null)
+            {
+                queuedChanges.Clear();
+                StopCoroutine(updateTilesRoutine);
+            }
+
+            lastUpdatedTimeStamp = Time.time;
+            visibleTilesDirty = true;
+            updateTilesRoutine = StartCoroutine(UpdateVisibleTiles());
+        }
+
+        private IEnumerator UpdateVisibleTiles()
+        {
+            //get current tiles
+            foreach (KeyValuePair<Vector2Int, Tile> tile in tiles)
+            {
+                if (tile.Value == null || tile.Value.gameObject == null)
+                    continue;
+
+                if (tile.Value.runningCoroutine != null)
```

Comment:
Moet je hier dan tile.Value.runningCoroutine niet actief op null zetten als je die stopt? IIRC wordt een Coroutine niet automatisch null. En als je dan een tweede keer dezelfde tile behandeld dan vraag ik mij af of dat goed gaat?